CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt

```diff
@@ -147,12 +149,19 @@ class GalleryFragment internal constructor() : Fragment() {
 
                             // If all photos have been deleted, return to camera
                             if (mediaList.isEmpty()) {
-                                fragmentManager?.popBackStack()
+//                                fragmentManager?.popBackStack()
+
+                                Navigation.findNavController(requireActivity(), R.id.fragment_container).navigate(
+                                        GalleryFragmentDirections.actionGalleryFragmentToCameraFragment2())
+
                             }
-                        }}
 
-                    .setNegativeButton(android.R.string.no, null)
-                    .create().showImmersive()
+                        }
+
+                        .setNegativeButton(android.R.string.no, null)
+                        .create().showImmersive()
+
+            } ?: Toast.makeText(activity, "No media file available", Toast.LENGTH_SHORT).show()
```

Comment:
I am not owner of this repository, but I think that it is good practice to avoid hardcoded texts. What about extracting this to R.string.no_media_file_available`?

Reply:
Yes it can be. I'm creating another pull for it. also check issue #134 solution please